sponsors.md

```diff
@@ -51,14 +51,19 @@ Thank you to NETWAYS, who provide servers for our continuous integration system.
 
 ### JetBrains
 
-<div class="lead text-left"><a href="http://www.jetbrains.com/ruby"></a><img src="/static/images/sponsors/jetbrains.svg" height="125px"/></div>
+<div class="lead text-left"><a href="http://www.jetbrains.com/ruby"><img src="/static/images/sponsors/jetbrains.svg" height="125px"/></a></div>
 Thank you to JetBrains, who provide licenses for Rubymine/IntelliJ IDE.
 
 ### ATIX
 
-<div class="lead text-left"><a href="https://www.atix.de"></a><img src="/static/images/sponsors/atix.png" height="125px"/></div>
+<div class="lead text-left"><a href="https://www.atix.de"><img src="/static/images/sponsors/atix.png" height="125px"/></a></div>
 Thank you to ATIX, for their many ongoing contributions to the codebase.
 
+### DM
+
+<div class="lead text-left"><a href="https://www.dm.de"><img src="/static/images/sponsors/dm.jpg" height="125px"/></a></div>
+Thank you to DM, for their many ongoing contributions to the codebase.
```

Comment:
I'd prefer `dm-drogeriemarkt` here.

Reply:
Is that as shown, without caps?

Reply:
On second thought, why not just use `dm` (lowercase on purpose). :-)

Reply:
@timogoebel cool, updated